### PR TITLE
Bold Position Manager Header Titles TM-2028

### PR DESCRIPTION
**Pull request by SidelineCory24**
This makes the titles match "Last Updated" as the ticket request but does not make use of `font-weight: bold`.

Also, the ticket says to make the "Bid Count" text smaller. What should it be decreased to? `16px` is the standard size for it on cards.

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
> This makes the titles match "Last Updated" as the ticket request but does not make use of `font-weight: bold`.
> 
> Also, the ticket says to make the "Bid Count" text smaller. What should it be decreased to? `16px` is the standard size for it on cards.

It's ok that it doesn't use of `font-weight: bold` to achieve the desired outcome; still looks good!
"What should it be decreased to?" 🤔  good question. I think 15px looks good! If everyone hates it, they'll let us know at the demo lol

File: src/sass/_bureau.scss [hunk 1]
```diff
@@ -319,11 +322,6 @@
     }
   }
 
-  .bid-count-label {
-    margin-right: 0;
-    min-width: 225px;
-  }
-
```
Comment (SidelineCory24) [inline src/sass/_bureau.scss:326]: Messes up the intended design of the Bid Count.

Comment (elizabeth-jimenez) [reply src/sass/_bureau.scss:326]: this change only applies to the Position Details and I think we want to stick to the Bid Count in the Position Manager

image

**Comment by elizabeth-jimenez** [comment]
🎉  Besides the changes in Position Details, this looks ready to go!

**Review by elizabeth-jimenez — APPROVED** [review]
🍬 🍭 🍰 🍥 👍

**Pull request merged**